base/applications/kbswitch/kbswitch.c

```diff
@@ -8,7 +8,9 @@
  */
 
 #include "kbswitch.h"
-#include <imm.h>
+
+// TODO: Move this
+HRESULT WINAPI SHLoadRegUIStringW(HKEY hkey, LPCWSTR value, LPWSTR buf, DWORD size);
```

Comment:
Just include `shlwapi_undoc.h`, the function is already declared there.

Reply:
OK.